steps/AddLang.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TechTalk.SpecFlow;
+using NUnit.Framework;
+
+namespace MarsQA_1.Feature

```

Comment:
Isn't this class under the Steps folder instead of Feature folder?

Reply:
Yes, that make sense. But as they gave the Login.feature and login.cs file in same feature folder I got confused whether to keep it in a same folder or Create a new steps folder.